### DISCUSSION TITLE
Why are regions blocked?

### DISCUSSION THREAD

**Discussion by AnnualfireCHW** [IviriusCommunity/Rebound#8] [Q&A] [answered]
很令人愤怒，我在你的开源文件上发现你屏蔽了俄罗斯和中国这两个地区的国家，请你说明为何这么做？

It's infuriating, I found out on your open source file that you blocked countries in the region of Russia and China, could you please explain why you did this?
如果你是出于语言翻译或与之相应的兼容性问题可以考虑招募一些翻译的人员，但无论如何选择直接屏蔽这两个国家，并且不给出任何说明，这都是违反开源协议！
If you're looking for language translation or compatibility issues, consider recruiting some translators, but in any case choosing to block these two countries directly without giving any explanation, it's a violation of the open source license!

**Comment by xiaojiang233** [comment]
支持
Im support.

**Comment by IviriusMain** [comment]
Hello!
Blocking Rebound 11 in Russia is part of general sanctions as a result of the war in Ukraine.
About China, it's a temporary restriction until we can make sure Rebound 11 doesn't contain components that might not be allowed there.
These bans will be lifted in v0.0.3 ALPHA.

**Comment by AnnualfireCHW** [comment, marked as answer]
> Hello! Blocking Rebound 11 in Russia is part of general sanctions as a result of the war in Ukraine. About China, it's a temporary restriction until we can make sure Rebound 11 doesn't contain components that might not be allowed there. These bans will be lifted in v0.0.3 ALPHA.

首先，你的政治立场与我们无关，无论你支持哪个国家，都不应该把你的政治立场带入开源软件中！
其次，我不明白你说的“屏蔽关于中国的组件”，这是一个翻译问题，请你用更简洁明了的语言进行阐述。
另外，我不想也不该在 Github 上讨论国际政治话题，我们都在朝着开源的美好方向前进，不要毁了开源社区环境！
将自己的政治偏见或立场添加到软件中，甚至对其作出限制是非常愚蠢的行为！！！我希望您理解这一点，祝你有美好的一天

First of all, your political stance has nothing to do with us, and no matter which country you support, you should not bring your political stance into open source software!
Secondly, I don't understand what you said about "blocking the components about China", this is a translation problem, please elaborate in more concise and clear language.
In addition, I don't want to and shouldn't discuss international politics on Github, we are all moving in the beautiful direction of open source, don't ruin the open source community environment!
It's foolish to add your own political biases or positions to the software, or even limit them!! I hope you understand this and have a nice day

**Reply by Lamparter** [reply]
Hello @AnnualfireCHW!
I have had a discussion with Ivirius internally about these racist and illegal views. _(it is illegal to open-source a project in such a way that your political bias affects it)_ He has promptly decided to lift the ban.

(I am a student of Simplified Chinese, here is what I have said roughly translated into mandarin)
你好 @AnnualfireCHW！
我对Ivirius说了我觉得Rebound11在中国不是禁止。他说了“好“； 今天Rebound11不是在中国禁止了。

**Reply by AnnualfireCHW** [reply]
我明白了，感谢理解，你们的决定非常开明
我会尽我所能，这个项目添砖加瓦😄
I see, thank you for your understanding, your decision is very enlightened
I will do my best to contribute to this project
: )

**Comment by AnnualfireCHW** [comment]
抛开上述这些，我很欣赏您的项目，它对于统一windows 11的视觉效果有巨大改变并且这是一项辛苦的工作，这方面我们的立场是相同的
但是，加入自己的政治立场是否是一件合理的行为？
作出不合理的限制，是否应该在开源社区发生？
请你自行斟酌
Putting all of the above aside, I appreciate your project, it makes a huge difference to unify the visuals of Windows 11 and it's been a hard work, we are on the same page in this regard
But is it a reasonable course of action to join one's own political stance?
Should unreasonable restrictions occur in the open source community?
At your own discretion

**Comment by IviriusMain** [comment]
Region blocking has been lifted. Rebound 11 is now free to use anywhere in the world.

**Reply by AnnualfireCHW** [reply]
Good！
